x/posts/simulation/genesis.go

```diff
@@ -44,6 +44,10 @@ func RandomizeGenState(simState *module.SimulationState) {
 
 // randomPosts returns randomly generated genesis posts
 func randomPosts(r *rand.Rand, subspaces []subspacestypes.Subspace, accs []simtypes.Account, params types.Params) (posts []types.Post) {
+	if len(subspaces) == 0 {
+		return nil
+	}
```

Comment:
This and all the other checks inside the genesis simluation are to avoid tests failing when there are no subspaces present